project/src/components/card/card.tsx

```diff
@@ -0,0 +1,42 @@
+function Card(): JSX.Element {
+  const someLink  = '#';
+
+  return (
+    <article className="cities__place-card place-card">
+      <div className="place-card__mark">
+        <span>Premium</span>
+      </div>
+      <div className="cities__image-wrapper place-card__image-wrapper">
+        <a href={ someLink }>
+          <img className="place-card__image" src="img/apartment-01.jpg" width="260" height="200" alt="Place pict"/>
```

Comment:
```suggestion
          <img 
            className="place-card__image" 
            src="img/apartment-01.jpg" 
            width="260" 
            height="200" 
            alt="Place pict"
          />
```
И вот так лучше всего форматировать, так становится читабельней и понятней